phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexUpgradeToolIT.java

```diff
@@ -137,4 +141,33 @@ private void prepareForTest(Connection conn, String schemaName, String tableName
             }
         }
     }
+
+    @Test
```

Comment:
another test to see if defaults are getting passed when conf does not set anything.